### ISSUE TITLE
Logit visualiser

### ISSUE THREAD

**Issue by alan-cooney** [alan-cooney/CircuitsVis#7]
A logit visualizer, which shows text coloured by the prob/log-prob predicted for that token, and if you hover over a token it shows the top 5 logits/probs/log-probs (I was planning on trying to hack this together with your library this evening, thus my questions)

**Comment by alan-cooney** [comment]
Neel may be currently working on this

**Comment by neelnanda-io** [comment]
Done in #36
